k3d/test-gslb4.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: k3d.io/v1alpha2
+kind: Simple
+name: test-gslb4
+image: docker.io/rancher/k3s:v1.21.2-k3s1
+agents: 1
+network: k3d-action-bridge-network
+ports:
+- port: 83:80
+  nodeFilters:
+  - agent[0]
+- port: 446:443
+  nodeFilters:
+  - agent[0]
+- port: 9083:30090
+  nodeFilters:
+    - agent[0]
+- port: 5056:53/udp
+  nodeFilters:
+  - agent[0]
+options:
+  k3d:
+    disableLoadbalancer: true
+  k3s: # options passed on to K3s itself
+    extraServerArgs: # additional arguments passed to the `k3s server` command; same as `--k3s-server-arg`
+      - --no-deploy=traefik,servicelb,metrics-server
```

Comment:
good point, I wanted to do it actually, but didn't want to complicate the makefile even further. It's done now and it's not that bad. So that .gitignore ignores all `/k3d/test-gslb*.yaml` except 1,2 and 3